app/graphql/mutations/create_submission_mutation.rb

```diff
@@ -29,6 +29,7 @@ class CreateSubmissionMutation < Mutations::BaseMutation
     argument :location_state, String, required: false
     argument :medium, String, required: false
     argument :minimum_price_dollars, Integer, required: false
+    argument :myCollectionArtworkID, String, required: false
```

Comment:
@kiryl-zubarau Shall I add `myCollectionArtworkID` to [SubmissionType](https://github.com/artsy/convection/blob/dbeb77e9dfb1b94f1e35e1b04151150bb3dfaa65/app/graphql/types/submission_type.rb#L4) as well?

Reply:
Looks like it's not required for now.